tensorflow/stream_executor/dnn.h

```diff
@@ -2103,7 +2104,8 @@ class DnnSupport {
                       dnn::RnnMode rnn_mode, dnn::DataType data_type,
                       const dnn::AlgorithmConfig& algorithm_config,
                       float dropout, uint64 seed,
-                      ScratchAllocator* state_allocator) {
+                      ScratchAllocator* state_allocator,
+                      bool use_padded_io) {
```

Comment:
+@deven-amd for awareness. Interface change in StreamExecutor.

Reply:
@houtoms 

please update the `createRnnDescriptor` signatures in `rocm/rocm_dnn.[h,cc]` as well, in order to avoid breaking the `--config=rocm` build. 

thanks

Reply:
+@aaroey for awareness.

@houtoms as per review from @deven-amd this PR would break ROCm build, would you mind help amend the signature in `rocm/rocm_dnn.[h.cc]`? thanks.

Reply:
Sure. No problem. I just submitted the change.

Reply:
thank you @houtoms for the update.